data/modules/Assassination/Assassination.lua

```diff
@@ -99,8 +99,8 @@ local onChat = function (form, ref, option)
 		local sbody = ad.location:GetSystemBody()
 
 		form:SetMessage(string.interp(l.X_WILL_BE_LEAVING, {
-		  target    = ad.target,
-		  spaceport = sbody.name,
+		  target    = ad.targetsurname or ad.target,	-- If targetsurname is missing, then older version.
+		  spaceport = sbody.name,						-- Instead hand over previously used full name + title.
```

Comment:
Is that comment indented with space or tabs? github editor gives me the impression it's tabs?

Reply:
Fixed!